src/main/java/net/dv8tion/jda/internal/requests/WebSocketClient.java

```diff
@@ -345,7 +346,7 @@ protected synchronized void connect()
             throw new RejectedExecutionException("JDA is shutdown!");
         initiating = true;
 
-        String url = api.getGatewayUrl()
+        String url = resumeUrl != null ? resumeUrl : api.getGatewayUrl()
```

Comment:
```suggestion
        String url = (resumeUrl != null ? resumeUrl : api.getGatewayUrl())
```
pretty sure these need `(..)`

Reply:
Lol ya beat me to it.